tests/pyright/__snapshots__/test_typing.ambr

```diff
@@ -0,0 +1,58 @@
+# name: test_typing
```

Comment:
Would there be a way to inser a comment here explaining that this is the expected pyright result file when test_typing.py is scanned with pyright (or whatever is the true explanation, I might have understood incorrectly :) )